docs/benchmarks/in-memory-analytics.md

```diff
@@ -94,6 +93,36 @@ Finally, run the benchmark as the client to the Spark master:
                  --driver-memory 4g --executor-memory 4g \
                  --master spark://SPARK-MASTER-IPADDRESS:7077
 
+### Multi-node deployment with dedicated network
```

Comment:
This part should assume using the `host` network for the containers and not a dedicated docker network.

Reply:
@neo-apz I don't understand. The purpose of this PR is also to add a tutorial for people who want to run these containers within a dedicated network. Why do you say `not a dedicated docker network`?